src/main/groovy/life/qbic/datamodel/dtos/business/Offer.groovy

```diff
@@ -132,6 +136,7 @@ class Offer {
          */
         double totalPrice
         double netPrice
+        double totalDiscountPrice
```

Comment:
the official term would be `amount` for each price. So here I would suggest `discountAmount`
https://www.invoiceberry.com/free-pdf-invoice-templates

Reply:
But then totalPrice would also not be correct, right

Reply:
What about `appliedTotalQuantityDiscount`?

Reply:
I would stay with the naming scheme we already have. At least for me its clear what it means. If we want to stick to a naming schema (not include the price) we should adapt this for the other variables as well (totalPrice, netPrice,..)